docs/fsharp/style-guide/formatting.md

```diff
@@ -100,42 +100,52 @@ If you have a long function definition, place the parameters on new lines and in
 
 ```fsharp
 module M =
-    let LongFunctionWithLotsOfParameters(aVeryLongParam: AVeryLongTypeThatYouNeedToUse)
-                                        (aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse)
-                                        (aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse)
-                                        =
-        // ... the body of the method follows
+    let LongFunctionWithLotsOfParameters
+        (aVeryLongParam: AVeryLongTypeThatYouNeedToUse)
+        (aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse)
+        (aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse)
+        =
+            // ... the body of the method follows
 ```
 
 This also applies to members, constructors, and parameters using tuples:
 
 ```fsharp
 type TM() =
-    member _.LongMethodWithLotsOfParameters(aVeryLongParam: AVeryLongTypeThatYouNeedToUse,
-                                            aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse,
-                                            aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse) =
-        // ... the body of the method
-
-type TC(aVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
+    member _.LongMethodWithLotsOfParameters
+        (
+            aVeryLongParam: AVeryLongTypeThatYouNeedToUse,
+            aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse,
+            aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse
+        ) =
+            // ... the body of the method
+
+type TC
+    (
+        aVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
         aSecondVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
-        aThirdVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse) =
-    // ... the body of the class follows
+        aThirdVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse
+    ) =
+        // ... the body of the class follows
```

Comment:
Same reasoning as previous comment (can't add a GH suggestion due to the deleted line, is that deletion intentional?)

```f#
type TC =
    (aVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
     aSecondVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
     aThirdVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse) =
```

Reply:
Please disregard this as per [this comment](https://github.com/dotnet/docs/pull/21690/files#r529190027)